rhtconsulting/rhc-ose
# Updated default values to use OSE v3.1

#### What does this PR do?

This PR changes the default values to use OSEv3.1 instead of OSEv3.0
#### How should this be manually tested?

Install OSE with default values - observe that 3.1 gets installed and everything is operational after installation.

```
./osc-provision --num-nodes=2 --key=<key_name>
```
#### Is there a relevant Issue open for this?

N/A
#### Who would you like to review this?

/cc @etsauer
